relayer/crates/starknet-chain-components/src/impls/queries/client_state.rs

```diff
@@ -57,10 +58,10 @@ where
         let client_id_seq = client_id
             .as_str()
             .rsplit_once('-')
-            .expect("valid client id")
+            .ok_or_else(|| Chain::raise_error("valid client id"))?
             .1
             .parse::<u64>()
-            .expect("valid sequence");
+            .map_err(|_| Chain::raise_error("valid sequence"))?;
```

Comment:
The error should probably be "invalid" instead of "valid". We could also use `CanRaiseError<ParseIntError>` for this case.

Reply:
nice catch. fixed at feb5c16